### PR TITLE
small fix to display webview within safe area

### PR DESCRIPTION
**Pull request by HarrisonHough**
This PR has updates that ensures the webview will fit within the safe area so the RPM website buttons are clickable.

### CODE REVIEW

**Review by chikul — APPROVED** [review]

File: Example-iOS-Swift/WebViewController.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ class WebViewController: UIViewController, WKScriptMessageHandler {
         let script = WKUserScript(source: source, injectionTime: .atDocumentEnd, forMainFrameOnly: false)
         config.userContentController.addUserScript(script)
         config.userContentController.add(self, name: "iosListener")
-        webView = WKWebView(frame: UIScreen.main.bounds, configuration: config)
+        webView = WKWebView(frame: .zero, configuration: config)
```
Comment (srcnalt) [inline Example-iOS-Swift/WebViewController.swift:31]: wow this `.zero` so weird, what it do?

**Review by srcnalt — APPROVED** [review]
Approved

**Review by tiivik — APPROVED** [review]

File: Example-iOS-Swift/WebViewController.swift (hunk 1 above)
Comment (HarrisonHough) [reply Example-iOS-Swift/WebViewController.swift:31]: Well the value passed in here as actually irrelevant as the size is controlled by the parent view controller. `.zero` creates a frame of size 0. I thought making it `.zero` would make it more obvious for peoples.

Comment (srcnalt) [reply Example-iOS-Swift/WebViewController.swift:31]: So you can put dot before variables in swift :S

**Pull request merged**